package.json

```diff
@@ -1,39 +1,146 @@
 {
-  "name": "ep-store-monorepo",
-  "private": true,
+  "name": "@elasticpath/ref-store",
   "version": "3.1.0",
   "main": "index.js",
   "license": "MIT",
-  "workspaces": {
-    "packages": [
-      "app",
-      "components",
-      "cortexjs",
-      "storybook"
-    ],
-    "nohoist": [
-      "**/webpack-dev-server",
-      "**/babel-loader"
-    ]
-  },
   "dependencies": {
-    "aws-amplify": "^1.1.29",
-    "aws-amplify-react": "^2.3.9",
-    "core-js": "^2.6.9",
-    "lerna": "^3.14.1",
+    "@babel/core": "^7.7.7",
+    "@elasticpath/store-components": "^1.1.48",
```

Comment:
We should remove the dependancy to the `@elasticpath/store-components` now that we are consuming the components as code?

Reply:
Removed